Stripe/StripeiOS/Source/STPAddCardViewController.swift

```diff
@@ -391,33 +395,37 @@ public class STPAddCardViewController: STPCoreTableViewController, STPAddressVie
     }
 
     func setUpCardScanningIfAvailable() {
-        if !STPCardScanner.cardScanningAvailable || configuration?.cardScanningEnabled != true {
-            return
+        if #available(macCatalyst 14.0, *) {
+            if !STPCardScanner.cardScanningAvailable || configuration?.cardScanningEnabled != true {
+                return
+            }
+            let scannerCell = STPCardScannerTableViewCell()
+            self.scannerCell = scannerCell
+            
+            let cardScanner = STPCardScanner(delegate: self)
+            cardScanner.cameraView = scannerCell.cameraView
+            self.cardScanner = cardScanner
+            
+            cardHeaderView?.buttonHidden = false
+            cardHeaderView?.button?.setTitle(
+                String.Localized.scan_card_title_capitalization,
+                for: .normal
+            )
+            cardHeaderView?.button?.addTarget(
+                self,
+                action: #selector(scanCard),
+                for: .touchUpInside
+            )
+            cardHeaderView?.setNeedsLayout()
         }
-        let scannerCell = STPCardScannerTableViewCell()
-        self.scannerCell = scannerCell
-
-        let cardScanner = STPCardScanner(delegate: self)
-        cardScanner.cameraView = scannerCell.cameraView
-        self.cardScanner = cardScanner
-
-        cardHeaderView?.buttonHidden = false
-        cardHeaderView?.button?.setTitle(
-            String.Localized.scan_card_title_capitalization,
-            for: .normal
-        )
-        cardHeaderView?.button?.addTarget(
-            self,
-            action: #selector(scanCard),
-            for: .touchUpInside
-        )
-        cardHeaderView?.setNeedsLayout()
     }
 
     @objc func scanCard() {
-        view.endEditing(true)
-        isScanning = true
-        cardScanner?.start()
+        if #available(macCatalyst 14.0, *) {
```

Comment:
Should we add this to the method signature instead of the impl? From a cursory look at the PR, it looks like it's only called from within an availability block anyway. Maybe that's incompatible with `addTarget(_:action:for:)`?

Reply:
Yep! Just clicked the wrong autofix option.